### PR TITLE
98 fill landing page real data

### PR DESCRIPTION
**Pull request by Saeed99Madi**

### CODE REVIEW

**Comment by dev-mohmmed-alshorafa** [comment]
in Our Services sections give gap between the img and the service

File: client/public/index.html [hunk 1]
```diff
@@ -1,53 +1,30 @@
...
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
-    />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <!--
+
+<head>
+  <meta charset="utf-8" />
+  <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```
Comment (dev-mohmmed-alshorafa) [inline client/public/index.html:6]: delete this and add paws icon

File: client/src/components/Header/Actions/FireCall/index.tsx [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
 import { EmergencyAvatar, EmergencyPhoneIcon } from '../../components.styled';
 
-export const FireCall = () => (
-  <EmergencyAvatar>
-    <EmergencyPhoneIcon />
-  </EmergencyAvatar>
-);
+export const FireCall = () => {
+  return (
+    <NavLink to="/emergency">
```
Comment (dev-mohmmed-alshorafa) [inline client/src/components/Header/Actions/FireCall/index.tsx:6]: should be add case when dosn't singIn(Show login form)

File: client/src/components/Header/Actions/UserTicket/index.tsx [hunk 3]
```diff
@@ -10,10 +11,12 @@ type Props = {
...
-    <UserAvatar alt="Reem" src={user?.avatar} />
-    <UserNameText gutterBottom variant="h5">
-      {user?.name}
-    </UserNameText>
-  </UserTicketBox>
+  <NavigationAnchor to={`profile/${user?.id}`}>
+    <UserTicketBox>
+      <UserAvatar alt="Reem" src={user?.avatar} />
+      <UserNameText gutterBottom variant="h5">
+        {user?.name}
```
Comment (dev-mohmmed-alshorafa) [inline client/src/components/Header/Actions/UserTicket/index.tsx:18]: give the name praimary color

**Review by dev-mohmmed-alshorafa — CHANGES_REQUESTED** [review]

**Comment by dev-mohmmed-alshorafa** [comment]
when i press in signin ,it show me signup form

**Review by Amoodaa — APPROVED** [review]

**Pull request merged**